docs/catalog/cloud/match-slo.md

```diff
@@ -0,0 +1,50 @@
+---
+version: 1.0
+submitted_by: danielmeixner
+published_date: TBD
+category: cloud
+tags: 
+ - [PATTERN_TAGS]
+---
+
+# Match your service level
```

Comment:
Consider changing to: Match your service level objectives to business needs

Reply:
done